requirements.txt

```diff
@@ -8,8 +8,9 @@ confuse>=2.0.0
 python-telegram-bot==13.15
 wikipedia
 nltk
-flask>=2.3.2
-flask-socketio
+Flask>=2.3.2
+Flask-SocketIO
+Flask-SQLAlchemy
```

Comment:
While I think we should consider adding SQLAlchemy to the project, it is not needed for the current version.

Reply:
Ok removed.